tests/test-traffic-cop.js

```diff
@@ -165,37 +167,49 @@ describe('mozilla-traffic-cop.js', function() {
             }
         });
 
-        it('should call setReferrerCookie by default', function() {
-            spyOn(Mozilla.TrafficCop, 'isRedirectVariation').and.returnValue(false);
-            spyOn(Mozilla.TrafficCop, 'generateRedirectUrl').and.returnValue('http://www.mozilla.com/en-US/?v=1');
-            spyOn(Mozilla.TrafficCop, 'setReferrerCookie').and.returnValue(true);
+        it('should call setReferrerCookie by default', function () {
```

Comment:
Issue (blocking): this test fails for me intermittently. I believe this is because the version of Karma we've now updated to runs tests in a random order (to enforce more robust test writing). I added this to the `describe` block which seems to fix the failure:

```javascript
afterEach(function() {
    cop.storeReferrerCookie = true;
});
```